### ISSUE TITLE
Note highlighting not working

### ISSUE THREAD

**Issue by nascent** [CoderLine/alphaTab#317]
# Question
Hi,

I just upgraded to the latest commit, 0.9.6.266 and I believe I have lost note highlighting.

I have set colors in my JavaScript settings, however while the song plays and presents well; I do not know how to highlight the notes as the song progresses.

In the past I have found that only the top staff of a grand stave highlights in any case, and it would be good if highlighting could occur on both staves if possible.

# Your environment (if applicable)

* Version used: (develop branch)
* Platform used: JavaScript
* Rendering engine used: SVG
* Browser Name and Version: Chrome/Firefox latest
* Operating System and version (desktop and mobile):

**Comment by nascent** [comment]
It was a css selector change.

**Comment by chris-saunders** [comment]
@Danielku15 @nascent 
What was changed in order to fix this? I am still running into the same issue where notes are not being highlighted on the bottom staff. It seems they never get the at-highlight class like the top staff does

**Comment by Danielku15** [comment]
All alphaTab elements have new CSS classes. I recommend that you have a look at the latest full sample and align it with your code: https://docs.alphatab.net/develop/assets/files/player/player.html

**Comment by chris-saunders** [comment]
@Danielku15 
I have already fixed the CSS selectors, and I still have the issue with the second staff of a grand staff.
I believe this is due to the `.at-highlight` class never being assigned to the second staff.
Further investigation shows that in the JS function `cursorUpdateBeat` the variable `className` is only ever the beats on the top staff.

I have created a JSFiddle to demonstrate the issue https://jsfiddle.net/rj63L2ey/5/
In the fiddle the top staff starts at b0, and the bottom staff starts at b15. `className`, which is used in the function to highlight elements only ever gets the beats for the top staff, 0-14.
